src/main/java/kr/allcll/backend/support/exception/AllcllErrorCode.java

```diff
@@ -17,7 +17,9 @@ public enum AllcllErrorCode {
     BASKET_NOT_FOUND("관심과목 정보가 존재하지 않습니다."),
 
     EXTERNAL_CONNECTION_TERMINATED("외부 서버와의 연결이 종료되었습니다."),
-    ;
+
+    SSE_EMITTER_NOT_FOUND("존재하지 않는 SSE 연결입니다"),
+    SSE_CONNECTION_DENIED("현재 SSE 연결을 할 수 없습니다.");
```

Comment:
```suggestion
    SSE_EMITTER_NOT_FOUND("존재하지 않는 SSE 연결입니다"),
    SSE_CONNECTION_DENIED("현재 SSE 연결을 할 수 없습니다."),
    ;
```

위에 코멘트에 달아 두었는데, 이렇게 수정하면 어떨까요?

Reply:
아~ 쉼표가 있으면 세미콜론이 안올라오는군요
저도 이거 라인포맷팅하면 위로 올라오는거 불편했는데 꿀팁감사합니당